scripts/qa_crowbarsetup.sh

```diff
@@ -3867,6 +3867,22 @@ function oncontroller_testsetup()
         fi
     fi
 
+    if [[ $want_s390 ]] ; then
+        if ! glance_image_exists xcat-sle11-s390x ; then
+            curl -s \
+                http://$clouddata/images/xCAT-SLE11SP3.img | \
+                openstack image create --public --property hypervisor_type=zvm \
```

Comment:
This is inconsistent with the sles12sp1 image. It was in http://$clouddata/images/s390x/...